src/gobupload/compare/main.py

```diff
@@ -135,7 +139,62 @@ def meets_dependencies(storage, msg):
     return True
 
 
-def _process_compare_results(storage, model, results, stats):
+def _get_modify_current_entities(storage: GOBStorageHandler, chunk: list[Row]) -> dict[str, dict]:
```

Comment:
Voor de laatste `dict` in het return type zou je nog de type parameters kunnen toevoegen.

Reply:
done